### ISSUE TITLE
how to use the whole Standard and overwrite one rule

### ISSUE THREAD

**Issue by sinkcup** [slevomat/coding-standard#1325]
I try to use the whole Standard and overwrite one rule `DeclareStrictTypes.spacesCountAroundEqualsSign`, but failed:

```xml
<?xml version="1.0"?>
<ruleset name="PSR12-Slevomat">
  <exclude-pattern>/vendor/</exclude-pattern>
  <arg name="extensions" value="php" />

  <!-- Include the whole PSR12 standard -->
  <rule ref="PSR12" />

  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/ruleset.xml">
    <exclude name="SlevomatCodingStandard.TypeHints.DeclareStrictTypes" />
  </rule>

  <config name="installed_paths" value="../../slevomat/coding-standard" />
  <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
    <properties>
      <property name="declareOnFirstLine" value="false" />
      <property name="linesCountBeforeDeclare" value="1" />
      <property name="linesCountAfterDeclare" value="1" />
      <property name="spacesCountAroundEqualsSign" value="0" />
    </properties>
  </rule>
</ruleset>
```

**Comment by kukulich** [comment]
https://github.com/slevomat/coding-standard#choose-which-sniffs-to-run

**Comment by sinkcup** [comment]
I known how to choose some rules, but I want use the whole standard, maybe the default value should be a recommended way.

**Comment by mabar** [comment]
Some sniffs are exact opposite, using all of them wouldn't work, you have to use a preconfigured ruleset. Some standards may not be compatible with other standards (I tried and failed), imho it's easier to go rule-by-rule. 

Of course, you don't have to, already done rulesets are available, e.g.:
- https://github.com/orisai/coding-standard-php - includes almost every sniff I found, extremely strict, not PSR-12 compatible (uses tabs)
- https://github.com/doctrine/coding-standard - not as strict, but PSR-12 compatible and highly popular

**Comment by roslov** [comment]
I also have my standard with a lot of Slevomat rules set up:
https://github.com/roslov/psr12ext
